repair/row_level.cc

```diff
@@ -2449,7 +2524,10 @@ future<> repair_service::uninit_ms_handlers() {
         ms.unregister_repair_row_level_stop(),
         ms.unregister_repair_get_estimated_partitions(),
         ms.unregister_repair_set_estimated_partitions(),
-        ms.unregister_repair_get_diff_algorithms()).discard_result();
+        ms.unregister_repair_get_diff_algorithms(),
+        ser::partition_checksum_rpc_verbs::unregister_repair_update_system_table(&ms),
```

Comment:
Btw, you can also use `ser::partition_checksum_rpc_verbs::unregister(&ms)` to avoid listing all verbs here.

Reply:
Thanks. I was confused by the unregister() generated  before, but now I know what it is. However, since we have to call register for each verb, I think it is better to be call unregister for each verb too for the sake of symmetry.

Reply:
Ok, I don't mind either way. It's trivial to change later, if we decide to do so.